add-graphics-renderer/src/main/java/com/esri/arcgisruntime/sample/addgraphicsrenderer/MainActivity.java

```diff
@@ -97,7 +99,6 @@ private void addGraphicsOverlay(){
         SimpleRenderer simpleRenderer = new SimpleRenderer();
         // create point symbol
         SimpleMarkerSymbol pointSymbol = new SimpleMarkerSymbol(yellow, 30, SimpleMarkerSymbol.Style.SQUARE);
-        pointSymbol.setOpacity(0.5f);
```

Comment:
Opacity is now at the RgbColor level - if the sample previously used a semi-transparent symbol by setting opacity, then  best change this to set an equivalent opacity in the RgbColor alpha value.